addons/addon-user-id-upgrade/packages/user-id-upgrade-post-deployment/lib/data/study-permissions-transforms.js

```diff
@@ -65,6 +65,7 @@ module.exports = [
   addUid,
   updateId,
   listValue('readonlyUsers'),
+  listValue('readwriteUsers'),
```

Comment:
There is no need to make any update to the user-id-upgrade addon.

Reply:
True, since it wasn't available before anyways.